website/src/pages/tutorial.md

```diff
@@ -0,0 +1,523 @@
+## Introduction
+
+In this hands-on tutorial, we will take a look at how we can use Yew to build web applications.
+**Yew** is a modern [Rust](https://www.rust-lang.org/) framework for building front-end web apps using [WebAssembly](https://webassembly.org/).
+Yew encourages a reusable, maintainable, and well-structured architecture by leveraging Rust's powerful type system.
+A large ecosystem of community-created libraries (or crates?) provide components that work out-of-the-box and
+commonly-used patterns such as state management.
+
+Using WebAssembly also allows us to take advantage of numerous crates available on [crates.io](https://crates.io)
+
+### What we are going to build
+
+Rustconf is an intergalactic gathering of the Rust community that happens annually.
+Rustconf 2020 had a plethora of talks that provided a good amount of information.
+In this hands on tutorial, we will be building a web application to help fellow Rustaceans
+have overview of the talks, and watch them all from one page.
+
+## Setting up
+
+### Prerequisites
+
+To get started, let's make sure we have an up-to-date development environment.
+We will need the following tools:
+- [Rust](https://www.rust-lang.org/).
+- [`trunk`](https://trunkrs.dev/)
+
+This tutorial also assumes you're already familiar with Rust.
+
+### Setting up the project
+
+First, create a new cargo project:
+
+```bash
+cargo new yew-app
+cd yew-app
+
+To verify the Rust environment is set up properly, run the initial project using the cargo build tool. 
+After output about the build process, you should see the expected "Hello, world!" message.
+
+```bash
+cargo run
+```
+
+## Our first static page
+
+To convert this simple command line application to a basic Yew web application, a few changes are needed.
+Update the files as follows:
+
+```toml title="Cargo.toml" {7}
+[package]
+name = "yew-app"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
++ yew = { git = "https://github.com/yewstack/yew/" }
+```
+
+```rust title="src/main.rs"
+use yew::prelude::*;
+
+#[function_component(App)]
+fn app() -> Html {
+    html! {
+        <h1>{ "Hello World" }</h1>
+    }
+}
+
+fn main() {
+    yew::start_app::<App>();
+}
+```
+
+Now, let's create and `index.html` at the root of the project.
+
+```html title="index.html"
+<!DOCTYPE html>
+<html lang="en">
+<head>
+</head>
+<body>
+
+</body>
+</html>
+```
+
+### Start the development server
+
+Run the following command to build and serve the application locally.
+
+```bash
+trunk serve
+```
+
+Trunk will helpfully rebuild your application if you modify any source files.
```

Comment:
Trunk will watch the project directory and helpfully rebuild your application if you modify any source files.